tests/unit/AoAlert.spec.js

```diff
@@ -55,4 +55,13 @@ describe('Alert', () => {
     jest.runAllTimers()
     expect(alert.emitted()['update:showAlert']).toBeTruthy()
   })
+
+  it('is dismissible', () => {
```

Comment:
this is only testing that the button is there, could we test that closeAlert is emitted on click, similar to the test above?

Reply:
Can do